### PR TITLE
add video_game content type.

### PR DESCRIPTION
**Pull request by blancoj**

### CODE REVIEW

File: spec/chipmunk/bagger/video_game_spec.rb
Comment (botimer) [inline spec/chipmunk/bagger/video_game_spec.rb:10]: We will need to set up a sample bag that conforms for this test to be meaningful.

File: README.md
Comment (botimer) [inline README.md:62]: This is the wrong profile. We should decide whether we publish this anywhere or just refer to the file on disk.

**Review by botimer — CHANGES_REQUESTED** [review]
This PR looks pretty close. Please see #26 and rebase once that is merged so the tests can be run easily. Then we will want to fix up the few problems here.

File: spec/chipmunk/bagger/video_game_spec.rb
Comment (blancoj) [reply spec/chipmunk/bagger/video_game_spec.rb:10]: I added the fixtures we had for digital.

File: README.md
Comment (blancoj) [reply README.md:62]: not sure what profile to use.

File: spec/chipmunk/bagger/video_game_spec.rb
Comment (botimer) [reply spec/chipmunk/bagger/video_game_spec.rb:10]: Ahh, I see. We will want to have an actual example. It's pretty confusing to have the "video game" be a completely different type.

File: README.md
Comment (botimer) [reply README.md:62]: We added the real profile to the server repo (https://github.com/mlibrary/chipmunk/pull/112/files#diff-ef536aed197983d2176ef0b9787424477815dca573423e5d299109628590affc). We need it here, too, and can just mention or link the path to it within this repo.

File: spec/chipmunk/bagger/video_game_spec.rb
Comment (blancoj) [reply spec/chipmunk/bagger/video_game_spec.rb:10]: Noah, should I as Lance for an example?  Where do we get the example?

**Comment by botimer** [comment]
I suspect the best approach will be to get a real package and trim it back or adjust portions to fabricate an example that is appropriate for a fixture.

File: spec/chipmunk/bagger/video_game_spec.rb
Comment (botimer) [reply spec/chipmunk/bagger/video_game_spec.rb:10]: I suspect the best approach will be to get a real package and trim it back or adjust portions to fabricate an example that is appropriate for a fixture.

Comment (blancoj) [reply spec/chipmunk/bagger/video_game_spec.rb:10]: I just got a trimmed package that contains nothing private from Lance.  Will send it up with next pr.

Comment (botimer) [reply spec/chipmunk/bagger/video_game_spec.rb:10]: I guess this is OK. We don't check anything about the profile for any of the types here, just that the basic metadata is added by the bagger subclass. The server will validate the profile and reject any errors. We could add robustness here at any time.

**Review by botimer — APPROVED** [review]

**Pull request merged**
